providers/FeedProvider.tsx

```diff
@@ -91,6 +99,43 @@ const FeedProvider = ({ children }: { children: ReactNode }) => {
     [feed, filter],
   );
 
+  const playFeedId = async (feedIndex: number) => {
+    const embed = findValidEmbed(feed[feedIndex]);
+    const feedObj = feed[feedIndex];
+    const url = embed?.url;
+    if (url) {
+      const metadata = await fetchMetadata(url, feedObj);
+      if (metadata)
+        dispatch({
+          type: 'PLAY',
+          payload: { metadata, feedId: feedObj.id },
+        });
+    }
+  };
+
+  const handleNext = async () => {
+    if (isChannelChanged) {
+      playFeedId(0);
+      setIsChannelChanged(false);
+    }
+
+    const feedIndex = feed.findIndex((feedObj: SupabasePost) => feedObj.id === player.feedId);
+    if (feedIndex > -1) {
+      if (feedIndex + 1 < feed.length) {
+        playFeedId(feedIndex + 1);
+      }
+    }
```

Comment:
why not just put an `else` statement here?
- remove redundant `isChannelChanged` state variable 🚮 
- if `feedIndex <= -1` then `playFeedId(0);`